TheIvano-180294/Task06.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```

Comment:
will not work in python. This is notebook magic

Reply:
just removed it